doc/man1/openssl-dsa.pod.in

```diff
@@ -127,6 +127,9 @@ a public key.
 
 =head1 EXAMPLES
 
+Examples equivalent to these can be found in the documentation for the
+non-deprecated L<openssl-pkey(1)> tool.
```

Comment:
We consistently use "command" not "tool"

Reply:
Changed these over to _command_.

It doesn't appear to be consistent, both _utility_ and _tool_ are also used.  _Command_ is the most common, so it makes sense.

Reply:
336 vs 16 vs 25.  PR to make that 377 to 0 coming up :)

Reply:
Sounds good.

Are any other names used I wonder?